stl/src/syserror.cpp

```diff
@@ -218,6 +218,10 @@ _CRTIMP2_PURE unsigned long __CLRCALL_PURE_OR_CDECL _Winerror_message(
 }
 
 _CRTIMP2_PURE const char* __CLRCALL_PURE_OR_CDECL _Syserror_map(int _Errcode) { // convert to name of generic error
+    if (_Errcode == 0) {
+        return "success";
+    }

```

Comment:
I think we can remove this special-case code completely by adding a new first element to `_Sys_errtab` with `_Errcode == 0` and `_Name == "success"`.